gst/mqtt/ntputil.h

```diff
@@ -10,10 +10,15 @@
  * @author  Wook Song <wook16.song@samsung.com>
  * @bug     No known bugs except for NYI items
  */
+#ifndef __NNS_NTP_UTIL_H__
+#define __NNS_NTP_UTIL_H__
 
 #include <stdint.h>
 #include <time.h>
-
+#include <byteswap.h>
```

Comment:
This header file is not necessary so please remove it.

Reply:
Right. This header was include during development. Removed. Thanks.